IOMC/EventVertexGenerators/src/GaussEvtVtxGenerator.cc

```diff
@@ -95,3 +91,17 @@ void GaussEvtVtxGenerator::sigmaZ(double s) {
                                        << "Illegal resolution in Z (negative)";
   }
 }
+
+void GaussEvtVtxGenerator::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+  edm::ParameterSetDescription desc;
+  desc.add<double>("MeanXIncm", 0.0);
+  desc.add<double>("MeanYIncm", 0.0);
+  desc.add<double>("MeanZIncm", 0.0);
+  desc.add<double>("SigmaXIncm", 0.0);
+  desc.add<double>("SigmaYIncm", 0.0);
+  desc.add<double>("SigmaZIncm", 0.0);
+  desc.add<double>("TimeOffsetInns", 0.0);
+  desc.add<edm::InputTag>("src");
+  desc.add<bool>("readDB");
```

Comment:
Why not having a default also here?

Reply:
So far all the scenarios were inheriting from `VtxSmearedCommon`:
https://github.com/cms-sw/cmssw/blob/3b943c0dbbdf4494cd66064a5a147301f38af295/IOMC/EventVertexGenerators/python/VtxSmearedParameters_cfi.py#L12-L15

What would be the suggested default option to be added here? I have no strong opinion on this.

Reply:
> What would be the suggested default option to be added here? I have no strong opinion on this.

it's not yet fully clear if this is going to be the way to go (even if I have high hopes the reading from DB could be the default). I would leave it undecided for the time being.

Reply:
Fine by me.